app/uk/gov/hmrc/traderservices/views/partials/receipt.scala.html

```diff
@@ -0,0 +1,60 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.traderservices.models.DeclarationDetails
+@import uk.gov.hmrc.traderservices.models.UploadedFile
+@import uk.gov.hmrc.traderservices.views.CaseConfirmationViewContext
+
+@(caseReferenceId: String, context: CaseConfirmationViewContext, declarationDetails: DeclarationDetails, uploadedFiles:Seq[UploadedFile], generatedAt: String)(implicit messages: Messages)
+<div class="receipt">
+  <img class="receipt__crown-logo" src="@controllers.routes.Assets.versioned("images/hmrc.svg")" alt="HM Revenue and Customs">
+
+  <h3 class="govuk-heading-l">@messages("receipt.heading")</h3>
+
+  <div class="receipt__info-box">
+    <dl>
+      <dt>@messages("receipt.caseReferenceNumber")</dt>
```

Comment:
Why wrapping case reference number in a call to resolve message? There will be always no message

Reply:
A similar comment applies to some other uses of `@message`